src/hotspot/os/windows/gc/z/zMapper_windows.cpp

```diff
@@ -264,7 +264,7 @@ void ZMapper::split_placeholder(zaddress_unsafe addr, size_t size) {
     );
 
   if (!res) {
-    fatal_error("Failed to split placeholder", addr, size);
+    fatal_error("Failed to split placeholder", untype(addr), size);
```

Comment:
Please don't.

`untype` does an assertion check. We don't want that here, when constructing arguments for a fatal error message.